packages/assets/scss/00-base/_colors.scss

```diff
@@ -46,3 +45,8 @@ $c-bd-divider-dark:      $c-gray-dark !default;
 $c-bd-error:             $c-error !default;
 $c-bd-input:             $c-gray-light !default;
 $c-bd-input-dark:        $c-gray !default;
+
+// Box shadow
+$c-box-shadow-black:      rgba($c-black, 0.25) !default;
+$c-box-shadow-primary-40: rgba($c-primary, 0.4) !default;

```

Comment:
please see my comments earlier, I don't think think these colors are needed because I don't see a difference with the styles that reference these